users/templates/users/registration.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+
+    <div class="container">
+        <h3 class="mb-4">Registration</h3>
+        <div class="d-flex justify-content-center">
+            <div class="mx-auto col-10 col-md-8 col-lg-6 vh-100 mt-lg-3">
+                <form method="POST" action=".">
+                    {% csrf_token %}
```

Comment:
Missing non_field_errors
https://docs.djangoproject.com/en/5.0/ref/forms/api/#django.forms.Form.non_field_errors
https://stackoverflow.com/a/16215790 - it's and old one but still valid. It's a good practice to add non field error handling.